src/PIL/ImageTk.py

```diff
@@ -184,6 +184,7 @@ def paste(self, im, box=None):
         :param im: A PIL image. The size must match the target region.  If the
                    mode does not match, the image is converted to the mode of
                    the bitmap image.
+        :param box: Deprecated.
```

Comment:
I know this was just removed in https://github.com/python-pillow/Pillow/pull/6178 to prevent people using it, but I think we should keep it and simply say it is deprecated, partly to prevent a docs warning if nothing else.